package.json

```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "lint-watch": "watchexec -c -w app -w db -w utils \"next lint --fix\""
+    "lint-watch": "watchexec -c -w app -w db -w utils \"next lint --fix\"",
+    "prettier": "prettier --write app/**/**/*.ts db/**/*.ts utils/**/*.ts"
```

Comment:
```suggestion
    "prettier": "prettier --write app/**/*.ts db/**/*.ts utils/**/*.ts"
```

Reply:
With that syntax it was unable to find the file `app/api/update-session/route.ts` on my system 🤔  Any idea why? I use bash.

Reply:
ah well, bash option https://askubuntu.com/a/1287138

Reply:
I modified the command:

> If the path points to an existing directory, Prettier CLI recursively finds supported files in that directory.